project/Build.scala

```diff
@@ -673,16 +673,6 @@ object Build {
               )
             )
           )
-      },
-      Test / unmanagedSourceDirectories ++= {
-        val base = (Test / sourceDirectory).value
-        scalaVersionsDependendent(scalaVersion.value)(Seq.empty[File]) {
-          case (2, n) if n >= 12 =>
-            Seq(
-              base / "scala-2",
-              base / "scala-2.12+"
```

Comment:
It was useful until we had Scala 2.11 in the build, now it's basically the same as `scala` so all Scala versions, since `scala-2.13+` meant Scala 2.13 and Scala 3
So maybe let's just remove this dead code

Reply:
done